pkg/goGit/goGit.go

```diff
@@ -17,7 +14,7 @@ import (
 func UpdateLocalCopies(names []string, config *config.Config) {
 	for i := 0; i < len(names); i++ {
 		fmt.Printf("Value of i currently is: %v\n", i)
-		if i >= 3 {
+		if i >= 5 {
 			fmt.Println("we break here")
 			break
 		}
```

Comment:
Remove the debug print. and why the if condition? If only for development purposes, I would remove these

Reply:
Resolved in [76b3090](https://github.com/AntonSkrub/GoGit-Integration/pull/6/commits/76b3090a4275bf0725703d0684da0ca154dd884f)